shoop/core/suppliers/base.py

```diff
@@ -61,7 +61,7 @@ def get_orderability_errors(self, shop_product, quantity, customer):
         if stock_status.error:
             yield ValidationError(stock_status.error, code="stock_error")
         if shop_product.product.stock_behavior == StockBehavior.STOCKED:
-            if quantity >= stock_status.logical_count:
+            if quantity > stock_status.logical_count:
```

Comment:
There are two hard things in computer science: naming things, cache invalidation and off-by-one errors.